Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -2840,7 +2840,12 @@ void RenderLayer::paintSVGResourceLayer(GraphicsContext& context, const AffineTr
 
     auto localPaintDirtyRect = LayoutRect::infiniteRect();
 
-    auto* rootPaintingLayer = enclosingSVGRootLayer();
+    auto* rootPaintingLayer = [this] () {
```

Comment:
s/this/&/?

Reply:
Done. But I think "this" was enough since the only thing needed from the context was calling parent(). Anyway, I don' think there is a big difference in this case.